app/uk/gov/hmrc/apiplatformmicroservice/apidefinition/controllers/binders/package.scala

```diff
@@ -96,41 +93,34 @@ package object binders {
     }
   }
 
-  private def warnOnEmailId(id: DeveloperIdentifier): DeveloperIdentifier = id match {
-    case EmailIdentifier(_) => logger.warn("Still using emails as identifier"); id
-    case _ => id
-  }
-
-  implicit def developerIdentifierBinder(implicit textBinder: PathBindable[String]): PathBindable[DeveloperIdentifier] = new PathBindable[DeveloperIdentifier] {
-    override def bind(key: String, value: String): Either[String, DeveloperIdentifier] = {
+  implicit def userIdBinder(implicit textBinder: PathBindable[String]): PathBindable[UserId] = new PathBindable[UserId] {
+    override def bind(key: String, value: String): Either[String, UserId] = {
       for {
         text <- textBinder.bind(key, value)
-        id <- DeveloperIdentifier(value).toRight(s"Cannot accept $text as a developer identifier")
-        _ = warnOnEmailId(id)
+        id <- UserId.parse(value).toRight(s"Cannot accept $text as a developer identifier")
```

Comment:
should this now say 'user id' rather than 'developer identifier'?

Reply:
Would make sense.